### DISCUSSION TITLE
Upload failed csv

### DISCUSSION THREAD

**Discussion by dmedi** [supabase/supabase#4971] [Questions]
Hello, I don't know why I can't upload a csv to import data in my database.

I have this error:

"Table o was created but we encountered an error while inserting rows: error message: request entity too large error code: 413 error id: not defined

check your spreadsheet if there are any discrepancies"

Does anyone have any idea how I can fix this?

**Comment by awalias** [comment]
>request entity too large

how large is the file you're uploading? and how many rows?

> error id: not defined

just a guess but perhaps some records are missing a required field (id)?

**Comment by dmedi** [comment]
> how large is the file you're uploading? and how many rows?

Only 815 Ko, only 64 rows

> just a guess but perhaps some records are missing a required field (id)?

i do not see problem here :-/
